app/views/dashboard/index.html.erb

```diff
@@ -109,28 +109,12 @@
               </div>
             </div>
             <div class="box-body text-center float-center">
-              <span class="float-center text-center">
-                  <div class="dropdown">
-                    <%= dropdown_button "dropdownMenu1", { text: "New Inventory", size: "lg" } %>
-                    <ul class="dropdown-menu float-center" style="text-align:center;margin-left: 190px;" aria-labelledby="dropdownMenu1">
-                     <li class="center-block">
-                      <%= view_button_to donations_path, { text: "All Donations", type: '', size: "md" } %>
-                    </li>
-                     <li class="center-block">
-                      <%= new_button_to new_donation_path, { text: "New Donation", type: '', size: "md" } %>
-                    </li>
-                     <li class="center-block">
-                      <%#= new_button_to scale_donations_path, { text: "Repackaged Donation", type: '', size: "md" } %>
-                    </li>
-                    </ul>
-                </div>
-              </span>
+              <%= new_button_to new_donation_path, { text: "New Donation" } %>
```

Comment:
Sorry if I'm missing something here, but why did we remove the other buttons?

Reply:
i just replaced the buttons, i removed the drop downs and put the direct link of the action indicated in the label, just like armahillo told me.

Reply:
Ah, I think I see it now, sorry!

Reply:
Yup this is great, thank you! :100: :100: